src/lib/components/InputGroup/README.mdx

```diff
@@ -0,0 +1,253 @@
+---
+name: InputGroup
+menu: 'Layouts'
+route: /components/input-group
+---
+
+# InputGroup
+
+InputGroup visually groups related Button, SelectField and TextField elements together.
+
+import {
+  Playground,
+  Props,
+} from 'docz'
+import Icon from '../../../docs/_components/Icon'
+import {
+  Button,
+  InputGroup,
+  SelectField,
+  TextField,
+} from '../..'
+
+## Basic Usage
+
+To implement the InputGroup component, you need to import it first:
+
+```js
+import { InputGroup } from '@react-ui-org/react-ui';
+```
+
+And use it:
+
+<Playground>
+  {() => {
+    const [fruit, setFruit] = React.useState('apple');
+    const options = [
+      {
+        label: 'Apple',
+        value: 'apple',
+      },
+      {
+        label: 'Pear',
+        value: 'pear',
+      },
+      {
+        label: 'Cherry',
+        value: 'cherry',
+      },
+    ];
+    return (
+      <InputGroup label="Your favourite fruit">
+        <SelectField
+          label="Your favourite fruit"
+          onChange={(e) => setFruit(e.target.value)}
+          options={options}
+          value={fruit}
+        />
+        <TextField
+          label="Variety"
+          placeholder="Eg. Golden delicious"
+        />
+        <Button label="Submit" />
+      </InputGroup>
+    );
+  }}
+</Playground>
+
+See [API](#api) for all available options.
+
+## General Guidelines
+
+Use input group to group **related fields and actions** that a user can
+take. Input fields and buttons should not be grouped just to save space on the
+screen.
+
+### Shared Property
+
+You can set size property directly on InputGroup to be shared for all fields
+and buttons inside the group. This property is then passed over to individual
+elements. At the same time, it **cannot be overridden** on the fields' or
+buttons' level. While technically possible, from the design point of view it's
+undesirable to group elements of totally different types or sizes.
+
+## Sizes
+
+All existing field and button sizes are also available on the input group level:
+small, medium, and large.
+
+<Playground>
+  <InputGroup
+    label="Small size"
+    size="small"
+  >
+    <TextField label="Input" />
+    <Button label="Submit" />
+  </InputGroup>
+  <InputGroup label="Medium size">
+    <TextField label="Input" />
+    <Button label="Submit" />
+  </InputGroup>
+  <InputGroup
+    label="Large size"
+    size="large"
+  >
+    <TextField label="Input" />
+    <Button label="Submit" />
+  </InputGroup>
+</Playground>
+
+## Invisible Label
```

Comment:
I think the labels of children inputs can be forced to be hidden at all times, without any chance to override it on the input level.

Reply:
Thanks to the `inputGroupContext` the labels of children inputs are always hidden. This "Invisible Label" part refers to the group label. Or did I misunderstand your comment?

Reply:
OK, thanks for the explanation.